src/lib/emitters.rs

```diff
@@ -378,6 +392,7 @@ impl RenderDotfile for MappingStore<String> {
                            common);
             digraph.push(line);
         }
+        digraph.push(String::from("\t{ rank=same; FROM0; TO0; }\n"));
```

Comment:
Is this intentionally hard-coded between two nodes?

Reply:
For now, yes. At the moment we assume that the root node is always node zero. This isn't entirely a safe assumption, as the edit script generator can replace the root node. This needs some careful thought though (and a new PR) so I have raised #41

Reply:
OK.